### DISCUSSION TITLE
How to spy on Vue 3 SFC methods?

### DISCUSSION THREAD

**Discussion by steveljko** [vitest-dev/vitest#615] [General]
I can't figure out how to implement spy on Vue 3 component methods?

I tried everything, but still I can't access methods...

**Comment by sheremet-va** [comment]
Why would you spy on implementation tho? 

I recommend watching latest talk on that issue here: https://youtu.be/L_BK8zXGdBM

Also docs touch on that subject: https://vue-test-utils.vuejs.org/guides/#common-tips

**Reply by steveljko** [reply]
I'm still new in this testing world. I thought it would be a good idea to test if a certain method from a component is called, and then see if the UI changes.

I will watch this video as soon as possible to get a better understanding of testing.

Thank you!
